glommio/src/io/buffered_file.rs

```diff
@@ -5,7 +5,7 @@
 //
 
 use crate::{
-    io::{glommio_file::GlommioFile, read_result::ReadResult},
+    io::{glommio_file::GlommioFile, read_result::ReadResult, OpenOptions},
```

Comment:
if we're now using your work only through immutable files, why do we need this patch ?

Reply:
Strictly speaking, we don't need this anymore, yes. However I'm keen on keeping the change as it makes the API cleaner and more consistent. 

I'm okay with taking it out into a standalone PR if you think it would make this one cleaner (but since it will be rebased on master, I'm not sure what would be the point in doing so).